src/Reflex/Dom/SemanticUI/Common.hs

```diff
@@ -65,6 +70,9 @@ instance UiClassText UiColor where
 class UiHasColor a where
   uiSetColor :: UiColor -> a -> a
 
+instance (Reflex t, UiHasColor a) => UiHasColor (Dynamic t a) where
```

Comment:
Oh, so that _wasn't_ what you had in mind? 🤔 

Nice thing is that this same pattern would work with `Event` and `Behavior`.